code/modules/vehicles/speedbike.dm

```diff
@@ -520,7 +520,14 @@
 
 /obj/vehicle/space/speedbike/speedwagon/buckle_mob(mob/living/M, force = 0, check_loc = 1)
  	. = ..()
-		riding_datum = new/datum/riding/space/speedbike/speedwagon
+    	riding_datum = new/datum/riding/space/speedwagon
```

Comment:
This was to solve a merge conflict, I have no idea why it won't compile like that master does